config.yaml

```diff
@@ -170,6 +175,10 @@ options:
     type: string
     default: ''
     description: Publicly accessible URL of a Juju Environment Manager.
+  jem-version:
+    type: string
+    default: 'v1'
```

Comment:
You used quotes here but not for charmstore-version. I don't think they are needed.